tests/integration/traces/traces_integ_base.py

```diff
@@ -53,11 +55,38 @@ def get_traces_command_list(
             command_list += ["--start-time", start_time]
         if end_time:
             command_list += ["--end-time", end_time]
-        if unformatted:
-            command_list += ["--unformatted"]
+        if output:
+            command_list += ["--output", output]
         if tail:
             command_list += ["--tail"]
         if beta_features:
             command_list += ["--beta-features"]
 
-        return command_list
\ No newline at end of file
+        return command_list
+
+    @staticmethod
+    def get_basic_deploy_command_list(
+        stack_name=None,
+        template_file=None,
+        capabilities=None,
+        resolve_s3=False,
+    ):
+
+        command_list = [TracesIntegBase.base_command(), "deploy"]
+
+        if stack_name:
+            command_list = command_list + ["--stack-name", str(stack_name)]
+        if template_file:
+            command_list = command_list + ["--template-file", str(template_file)]
+        if capabilities:
+            command_list = command_list + ["--capabilities", str(capabilities)]
+        if resolve_s3:
+            command_list = command_list + ["--resolve-s3"]
+
+        return command_list
+
+    @staticmethod
+    def _method_to_stack_name(method_name):
+        """Method expects method name which can be a full path. Eg: test.integration.test_deploy_command.method_name"""
+        method_name = method_name.split(".")[-1]
+        return f"{method_name.replace('_', '-')}-{CFN_PYTHON_VERSION_SUFFIX}"
```

Comment:
I won't recommend this. If we need to change (or randomize) stack names, we need to change it in multiple places. I would recommend finding more generic approach so that all of these tests (deploy, sync, traces) can use the same method to get the stack name.

Reply:
Agree, let me figure a way to move this method to a utility class.

Reply:
updated in the latest commit.